books/free-programming-books-langs.md

```diff
@@ -1940,7 +1940,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [A Guide to Python's Magic Methods](https://github.com/RafeKettler/magicmethods) - Rafe Kettler
 * [A Practical Introduction to Python Programming](https://www.brianheinold.net/python/) - Brian Heinold (HTML, PDF, Exercises sources)
 * [A Whirlwind Tour of Python](https://jakevdp.github.io/WhirlwindTourOfPython/) - Jake VanderPlas
-* [An Introduction to Statistical Learning with Applications in Python](https://hastie.su.domains/ISLP/ISLP_website.pdf) - Gareth James, Daniela Witten, Trevor Hastie, Robert Tibshirani, Jonathan Taylor (PDF)
+* [An Introduction to Statistical Learning with Applications in Python](https://www.statlearning.com/) - Gareth James, Daniela Witten, Trevor Hastie, Robert Tibshirani, Jonathan Taylor (PDF)
```

Comment:
trailing slash no longer accpeted. For further information see [here](https://github.com/vhf/remark-lint-no-url-trailing-slash#remark-lint-no-url-trailing-slash)

change
```text
https://www.statlearning.com/
```
to
```text
https://www.statlearning.com
```

Reply:
unless you make the suggested change linter issue won't be solved